package.json

```diff
@@ -73,6 +73,7 @@
     "pg": "4.1.1"
   },
   "devDependencies": {
+    "async": "2.0.0-rc.3",
```

Comment:
thanks for hint. the difference between ```async.waterfall``` and sequence is that your sequence util helper expects that the functions in the array return a promise. setup and teardown are right now callback designed, but i changed it to support promise and callback usage. with this change i am able to use sequence.